kentikapi/internal/resources/alerting_test.go

```diff
@@ -85,11 +85,12 @@ func TestGetActiveAlerts(t *testing.T) {
 	alarmEndStr := "0000-00-00 00:00:00"
 	expected := []models.Alarm{
 		{
-			AlarmID:         82867908,
+			AlarmID:         "82867908",
 			RowType:         "Alarm",
 			AlarmState:      "ALARM",
-			AlertID:         15094,
-			TresholdID:      76518,
+			AlertID:         "15094",
+			MitigationID:    "0",
```

Comment:
```suggestion
			MitigationID:    nil,
```
`"mitigation_id": null,` response should be converted to `""` or nil models.ID

MitigationID should probably be exposed as optional value (using pointer) to be consistent with rest of library. So this should be nil.

Reply:
Done